lib/pub_sub/message.rb

```diff
@@ -11,20 +11,22 @@ def initialize(payload)
     end
 
     def process
-      validate_message!
+      begin
+        validate_message!
+      rescue PubSub::ServiceUnknown => e
```

Comment:
If we check for this case in `validate_message!` (which seems to be the right place), then we have no other good way to convey that information to `process`, which must successfully exit right after this failed validation. So, alternatively, I could test this case in `process` itself but then I'd be littering the class with checks whereas they really belong to `validate_message!`